src/extensionsIntegrated/CustomSnippets/main.js

```diff
@@ -40,8 +40,8 @@ define(function (require, exports, module) {
 
     const MY_COMMAND_ID = "custom_snippets";
     const PANEL_ID = "customSnippets.panel";
-    const MENU_ITEM_NAME = "Custom Snippets..."; // this name will appear as the menu item
-    const PANEL_MIN_SIZE = 100; // the minimum size more than which its height cannot be decreased
+    const MENU_ITEM_NAME = "Custom Snippets\u2026"; // this name will appear as the menu item
```

Comment:
Shouldnt this be in `Strings.js` for translation. Check if snippets panel is translated, ie all strings are added in strings.js.